DESCRIPTION

```diff
@@ -47,6 +47,7 @@ Suggests:
     osmextract,
     stplanr,
     dplyr,
-    rgdal
+    rgdal,
+    tmap
```

Comment:
A very handy package for the vignette :+1: for using it. I was being overly cautious before but truth is adding as Suggests does not affect 'installability'.